src/Progress/Progress.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useEffect, useRef, useState, forwardRef, memo} from 'react'
+import {View, StyleSheet, Animated, Dimensions} from 'react-native'
+
+interface IProgressProps {
+  /**
+   * Current percent of the Progress
+   * default 0, max 100
+   */
+  value: number
+  /**
+   * Defines height of Progress
+   * default 16
+   */
+  size?: number
+  /**
+   * Defines borderRadius of Progress
+   * default 0
+   */
+  borderRadius?: number
+  /**
+   * Defines color of Track Bar
+   * default #49BE25
+   */
+  filledTrackColor?: string
+  /**
+   * Defines background color of Progress
+   * default #E5E5E5
+   */
+  backgroundColor?: string
+}
+
+const {width: screenWidth} = Dimensions.get('screen')
+
+const Progress = (props: IProgressProps, ref?: any) => {
+  const {
+    value = 0,
+    size = 16,
+    borderRadius = 0,
+    filledTrackColor = '#49BE25',
+    backgroundColor = '#E5E5E5',
+  } = props
```

Comment:
can you change destructuring props for the component?

like this:
const Progress = ({value, size, ...}: IProgressProps, ref?: any) => {}

Reply:
Sure

Reply:
Done